3rd-party/class.jetpack-amp-support.php

```diff
@@ -82,274 +106,320 @@ function_exists( 'amp_is_canonical' ) // this is really just testing if the plug
 					( version_compare( AMP__VERSION, '1.0', '<' ) && self::has_amp_suffix() ) // after AMP 1.0, the amp suffix will no longer be supported
 				);
 
+			/**
+			 * Returns true if the current request should return valid AMP content.
+			 *
+			 * @since 6.2.0
+			 *
+			 * @param boolean $is_amp_request Is this request supposed to return valid AMP content?
+			 */
+			return apply_filters( 'jetpack_is_amp_request', $is_amp_request );
+		}
 		/**
-		 * Returns true if the current request should return valid AMP content.
-		 *
-		 * @since 6.2.0
-		 *
-		 * @param boolean $is_amp_request Is this request supposed to return valid AMP content?
+		 * Has AMP Suffix.
 		 */
-		return apply_filters( 'jetpack_is_amp_request', $is_amp_request );
-	}
-
-	static function has_amp_suffix() {
-		$request_path = wp_parse_url( $_SERVER['REQUEST_URI'], PHP_URL_PATH );
-		return $request_path && preg_match( '#/amp/?$#i', $request_path );
-	}
-
-	static function filter_available_widgets( $widgets ) {
-		if ( self::is_amp_request() ) {
-			$widgets = array_filter( $widgets, array( 'Jetpack_AMP_Support', 'is_supported_widget' ) );
+		static function has_amp_suffix() {
+			$request_path = wp_parse_url( $_SERVER['REQUEST_URI'], PHP_URL_PATH );
+			return $request_path && preg_match( '#/amp/?$#i', $request_path );
 		}
 
-		return $widgets;
-	}
+		/**
+		 * Filter Available Widgets.
+		 *
+		 * @param  [type] $widgets Widgets.
+		 * @return [type]          [description]
+		 */
+		static function filter_available_widgets( $widgets ) {
+			if ( self::is_amp_request() ) {
+				$widgets = array_filter( $widgets, array( 'Jetpack_AMP_Support', 'is_supported_widget' ) );
+			}
 
-	static function is_supported_widget( $widget_path ) {
-		return substr( $widget_path, -14 ) !== '/milestone.php';
-	}
+			return $widgets;
+		}
 
-	static function amp_disable_the_content_filters() {
-		if ( defined( 'WPCOM') && WPCOM ) {
-			add_filter( 'videopress_show_2015_player', '__return_true' );
-			add_filter( 'protected_embeds_use_form_post', '__return_false' );
-			remove_filter( 'the_title', 'widont' );
+		/**
+		 * Is Supported Widget.
+		 *
+		 * @param  string $widget_path Widget Path.
+		 */
+		static function is_supported_widget( $widget_path ) {
+			return substr( $widget_path, -14 ) !== '/milestone.php';
 		}
 
-		remove_filter( 'pre_kses', array( 'Filter_Embedded_HTML_Objects', 'filter' ), 11 );
-		remove_filter( 'pre_kses', array( 'Filter_Embedded_HTML_Objects', 'maybe_create_links' ), 100 );
-	}
+		/**
+		 * AMP Disable The Content Filters.
+		 *
+		 * @return [type] [description]
```

Comment:
You have a few `@return [type] [description]` entries in your PR. It would be good to move away from the default and specify a type and a description.